src/Spice86/ViewModels/MainWindowViewModel.cs

```diff
@@ -228,14 +235,15 @@ public void SetResolution(int width, int height) {
 
     [RelayCommand(CanExecute = nameof(IsEmulatorRunning))]
     private async Task DumpEmulatorStateToFile() {
-        //TODO: refactor this
         await _hostStorageProvider.DumpEmulatorStateToFile().ConfigureAwait(false);
     }
 
     [RelayCommand(CanExecute = nameof(CanPause))]
     public void Pause() => _pauseHandler.RequestPause("Pause button pressed in main window");
 
-    private void SetMainTitle() => MainTitle = $"{nameof(Spice86)} {Configuration.Exe}";
+    private void SetMainTitle(double instructionsPerMillisecond) {
+        MainTitle = $"{nameof(Spice86)} {Configuration.Exe} - {instructionsPerMillisecond:N0} cycles/ms";
```

Comment:
Slightly less jerky:
```suggestion
        MainTitle = $"{nameof(Spice86)} {Configuration.Exe} - cycles/ms: {instructionsPerMillisecond,7:N0}";
```